README.md

```diff
@@ -191,8 +191,11 @@ Note that order represents priority
 
 api-key: <api-key-generated-from-the-server>
 server-ip: https://localhost
+facts-service-url: amqp://guest:guest@localhost:5672
```

Comment:
I think we need to adjust this part.
The facts-service-URL depends on the configuration of RabbitMQ.

Usually we create a user with a password:
```
rabbitmqctl add_user trento_user trento_user_password
```
and a virtual host:
```
rabbitmqctl add_vhost vhost
```
This params are a must for the facts service url, so we need to adjust the example.

```facts-service-url: amqp://guest:guest@localhost:5672``` -> ```facts-service-url: amqp://guest:guest_password@localhost:5672/vhost```

or we use the same facts-service-url which is mentioned in the manual installation guide.
```
amqp://trento_user:trento_user_password@trento.example.com:5672/vhost
```